DigitalLearningSolutions.Web.Tests/Controllers/TrackingSystem/CourseSetup/ManageCourseControllerTests.cs

```diff
@@ -49,5 +49,51 @@ public void Index_returns_ManageCourse_page_when_appropriate_course_found()
             // Then
             result.Should().BeViewResult().WithDefaultViewName().ModelAs<ManageCourseViewModel>();
         }
+
+        [Test]
+        public void Edit_Course_Options_with_course_details_page_opens()
+        {
+            // Given
+            A.CallTo(() => courseDataService.GetCourseDetailsForAdminCategoryId(A<int>._, A<int>._, A<int>._))
+                .Returns(new CourseDetails());
+
+            // When
+            var result = controller.EditCourseOptions(1);
+
+            // Then
+            result.Should().BeViewResult().WithDefaultViewName().ModelAs<EditCourseOptionsViewModel>();
+        }
+
+        [Test]
+        public void Edit_Course_Options_page_when_coursed_details_are_not_updated()

```

Comment:
Typo 'coursed' here and on the test below.

Reply:
Sorry for typo. this has been corrected.

Reply:
I think you still need to push the commit with the fix.